### DISCUSSION TITLE
Different Sequences for a range sweep command line

### DISCUSSION THREAD

**Discussion by syed-ahmed** [facebookresearch/hydra#2142] [Q&A] [answered]
Hi,

Is there a way to have a different type of sequence for a range sweep? My use case is the following:
```python
func_params.dim_k=2,4,8,16,32,64,128,256,512,1024,2048,4096,8192,16384
```

**Comment by Jasha10** [comment]
I can't think of any better way than your example above.
There was a related feature request in the past: https://github.com/facebookresearch/hydra/issues/1944

**Comment by jieru-hu** [comment, marked as answer]
hi @syed-ahmed  with the upcoming Hydra 1.2 you'd be able to configure the search space in your config file https://hydra.cc/docs/next/tutorials/basic/running_your_app/multi-run/#sweeping-via-hydrasweeperparams

in your case you might be able to use a omegaconf resolver to define such a range in the config file. 

you can also check out other sweepers, i believe all of them support log scale.

**Reply by syed-ahmed** [reply]
Thank you!
